ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-client-source-interfaces.j2

```diff
@@ -0,0 +1,69 @@
+{# doc - ip client source interfaces #}
+{% if ip_ftp_client_source_interfaces is arista.avd.defined or ip_http_client_source_interfaces is arista.avd.defined or ip_ssh_client_source_interfaces is arista.avd.defined or ip_telnet_client_source_interfaces is arista.avd.defined or ip_tftp_client_source_interfaces is arista.avd.defined %}
+#### IP Client Source Interfaces
+
+{%     if ip_ftp_client_source_interfaces is arista.avd.defined %}
+
+#### IP FTP Client Source Interfaces
```

Comment:
Instead of a section per client I think we can collapse them all into a single table like:

| IP Client | VRF | Source Interface |
| --------- | --- | ---------------- |

Reply:
Done